src/app/profile/payment-methods/payment-method/forms/add-payment-method/modal.tpl.html

```diff
@@ -0,0 +1,86 @@
+<div class="modal-content give-modal-content">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+
+        <div class="col-xs-12">
+          <button ng-click="$ctrl.cancel()" type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+          </button>
+          <h4 class="modal-title border-bottom-small">Add Payment Method</h4>
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-xs-12">
+
+          <div class="alert alert-danger" role="alert" ng-if="$ctrl.error">
+            <p>{{$ctrl.error}}</p>
+          </div>
+
+          <div class="mb_x">
+            <div class="row">
+              <div class="col-md-6">
+                <a
+                  ng-class="{'on':$ctrl.paymentType == 'bankAccount'}"
+                  ng-click = "$ctrl.switchPaymentType('bankAccount')"
+                  class="tab-btn btn btn-lg btn-default btn-block">
+                  Bank Account
+                  <img src="/assets/img/icon-bank.png" alt="Bank Account Icon">
+                </a>
+              </div>
+              <div class="col-md-6">
+                <a
+                  ng-class="{'on':$ctrl.paymentType == 'creditCard'}"
+                  ng-click = "$ctrl.switchPaymentType('creditCard')"
+                  class="tab-btn btn btn-lg btn-default btn-block">
+                  Credit Card
+                  <img src="/assets/img/icon-cc.png" alt="Credit Card Icon">
+                </a>
+              </div>
+            </div>
+          </div>
+          <!-- bank account form -->
+          <div ng-if="$ctrl.paymentType === 'bankAccount'" class="tab-indiv cart-tab show">
+            <div class="row">
+              <div class="col-md-12 col-xs-12">
+                <checkout-bank-account submitted="$ctrl.submitted" on-save="$ctrl.onSave(success)"></checkout-bank-account>
+              </div>
+            </div>
+          </div>
+        </div><!-- bank account form -->
+        <!-- credit card form -->
+        <div ng-if="$ctrl.paymentType === 'creditCard'" class="tab-indiv cart-tab show">
+          <div class="row">
+            <div class="col-md-12 col-xs-12">
+              <checkout-credit-card submitted="$ctrl.submitted" on-save="$ctrl.onSave(success,error)"></checkout-credit-card>
+            </div>
+          </div>
+        </div><!-- credit card form -->
+        <!-- loading -->
+        <div class="loading-wrap" ng-if="$ctrl.submitted">
```

Comment:
Use the loadingOverlay component here, check the search page or home page login for an example.